content/en/docs/reference/using-api/cel.md

```diff
@@ -342,12 +342,12 @@ use the underscore escaping
 Examples on escaping:
 
 {{< table caption="Examples escaped CEL identifiers" >}}
-| property name | rule with escaped property name   |
-|---------------|-----------------------------------|
-| `namespace`   | `self.__namespace__ > 0`          |
-| `x-prop`      | `self.x__dash__prop > 0`          |
-| `redact__d`   | `self.redact__underscores__d > 0` |
-| `string`      | `self.startsWith('kube')`         |
+| property name | rule with escaped property name        |
+|---------------|----------------------------------------|
+| `namespace`   | `self.__namespace__ > 0`               |
+| `x-prop`      | `self.x__dash__prop > 0`               |
+| `redact__d`   | `self.redact__underscores__d > 0`      |
+| `string`      | `self.__{string}__.startsWith('kube')` |
```

Comment:
We don't escape string as field name. I don't understand this change..

Reply:
How about dropping the line:
```markdown
| `string`      | `self.startsWith('kube')`         |
```
entirely?

Reply:
I believe this would a shame as an example makes it clear how reserved keywords should be escaped.
Could we get feedback from someone who knows how CEL escape sequences work and make sure the example is correct?

Reply:
> https://github.com/kubernetes/website/pull/47382#discussion_r1707420686

@cici37 knows a thing or two!

Reply:
I'm in favor of having tests in code that match all examples given here. That way if there is any disagreement about how the system behaves, we have direct evidence of what the actual behavior is.